packages/synapse-react-client/src/lib/containers/discussion_forum/DiscussionThread.tsx

```diff
@@ -292,24 +290,16 @@ export function DiscussionThread(props: DiscussionThreadProps) {
         confirmButtonColor="info"
         confirmButtonText="Restore"
       />
-      <Modal
-        className="bootstrap-4-backport"
-        show={showSignInModal}
-        onHide={() => setShowSignInModal(false)}
-        animation={false}
-      >
-        <Modal.Header closeButton />
-        <Modal.Body>{SIGN_IN_TEXT}</Modal.Body>
-        <Modal.Footer>
-          <Button
-            onClick={() => setShowSignInModal(false)}
-            variant="contained"
-            className={SRC_SIGN_IN_CLASS}
-          >
-            Sign In
-          </Button>
-        </Modal.Footer>
-      </Modal>
+      <ConfirmationDialog
+        open={showSignInModal}
+        title="Sign In Required"
```

Comment:
Added a title to the modal to match the [DownloadLoginModal styling](https://github.com/Sage-Bionetworks/synapse-web-monorepo/pull/185#issue-1694605368) and prevent a large whitespace at the top of the MUI modal

main | feature
:---:|:---:
![main_DiscussionThread_LoggedOut](https://user-images.githubusercontent.com/26949006/236285553-dc86f749-ce38-44c3-ae0b-64dc845c6e31.png) | ![feature_DiscussionThread_LoggedOut](https://user-images.githubusercontent.com/26949006/236285537-8957261a-cbe6-4eb2-bc47-bc018ecdd0f5.png)